server/controllers/UserController.js

```diff
@@ -54,6 +56,8 @@ class UserController {
 
   async delete(userId) {
     try {
+      await this.tripController.deleteTripsByUserId(userId)
```

Comment:
are we gonna add a "delete account" button or is this for dev purposes? if it's the latter mayb we can just run a mongo query for that?

Reply:
have to ask @AndyLiang1 about that. imo, i think its fine to have those extra functions in controllers (eg. if we do decide to add delete account in the future) since they are not exposed via a route anyways

Reply:
yepps true